src/PlayerAccount/PlayerAccount.php

```diff
@@ -114,6 +114,30 @@ public function incident(int $userId, string $incident)
         return $this->post($endpoint, $payload);
     }
 
+    /**
+     * Updates a player's attributes
+     *
+     * @param int $userId
+     * @param array $attributes
+     * @param int|null $adminUserId
+     *
+     * @return Response|RequestInsurance
+     *
+     * @throws PlayerAccountRequestFailedException|BindingResolutionException
+     */
+    public function update(int $userId, array $attributes, ?int $adminUserId = null)
+    {
+        $payload = $attributes;
+
+        if($adminUserId !== null) {
+            $payload = array_merge($payload, ['admin_user_id' => $adminUserId]);
```

Comment:
Tænker dette er både hurtigere og nemmere at læse end en array_merge

```suggestion
            $payload['admin_user_id'] = $adminUserId;
```